kolibri/plugins/device/assets/src/views/ManageContentPage/NewBadge.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+
+  <span
+    class="new-label"
+    :style="{ color: $themeTokens.textInverted, backgroundColor: $themeTokens.success }"
+  >
+    {{ label }}
+  </span>
+
+</template>
+
+
+<script>
+
+  import commonDeviceStrings from '../commonDeviceStrings';
+
+  export default {
+    name: 'NewBadge',
+    mixins: [commonDeviceStrings],
+    props: {
+      label: {
+        type: String,
+        required: true,
+      },
+    },
+  };
+
+</script>
+
+
+<style>
```

Comment:
```suggestion
<style lang="scss" scoped>
```

Could you convert these to scoped styles? You will also see that the `NewBadge` component is still inheriting styles from the parent components. If you make them scoped, they lose the styling:

![image](https://user-images.githubusercontent.com/10248067/111368330-fa513b80-8652-11eb-820e-3d84d7615a69.png)

Reply:
Done!

Reply:
It's not enough to just change the scoping. You'll need to transfer the styles from the `new-badge` classes into the component, and also make sure that the styling is responsive. For example, in `WithSizeAndOptionslvue`, the `.new-label.panel-sm` styles would have been applied before you extracted `NewBadge`

Reply:
It might be good enough to just add another `new-label` class to the `NewBadge` component like this `<NewBadge class="new-label">`. Then both the styles from the parent and from within the component will be applied.

Reply:
I am sorry but I'm not able to reproduce this. I am getting a perfectly aligned component (even without the scoping).
![Screenshot from 2021-03-17 01-58-38](https://user-images.githubusercontent.com/34203227/111375538-61f39f00-86c4-11eb-9026-50cd97747b47.png)

The `.new-label.panel-sm` style applies too.
![Screenshot from 2021-03-17 01-58-50](https://user-images.githubusercontent.com/34203227/111375685-910a1080-86c4-11eb-88b9-623f696bdfd6.png)

Is there something I am missing?